### DISCUSSION TITLE
Mod support

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#57] [Ideas]
how do you handle your "AppID_upgrade.ps1" mod files ?

**Comment by KnifMelti** [comment]
I'll write it up when I have more time...

**Reply by KnifMelti** [reply]
Done.
Now I'll have to test my function/solution (using Winget-Install custom parameters to first install an older version) when upgrading...

**Reply by Romanitho** [reply]
it's merged ;)

**Reply by KnifMelti** [reply]
Nice!
I'll complete my part in WAU as soon as I can/have the time/mana to do it.

**Reply by KnifMelti** [reply]
Completed now in my fork - extensive testing begins...

**Reply by KnifMelti** [reply]
Pull requested.
